### PR TITLE
Feature/123

### PR DESCRIPTION
**Pull request by MichaelXian**
Resolving problem 3 in #122. The results data now contains the same data as database schema, and the table-row displays the new data properly (except the ones that were not laid out in the wireframe, which are not displayed at all currently.)

### CODE REVIEW

File: client/src/components/ResultRow.js [hunk 1]
```diff
@@ -31,9 +31,9 @@ class ResultRow extends Component {
     let arrow = '>';
     let originalPosting = '';
     let applyButton = '';
-    let { location } = this.props.result;
+    let location = this.props.result.city + ", " + this.props.result.state + " " + this.props.result.country;
     if (this.state.selected) {
-      location = ''; // Hide things that shouldnt be shown when selected, show things that should be shown
+      location = ''; // Hide things that shouldnt be shown whenconstructor(title, link, description, excerpt, city, state, country, latitude, longitude, companyName, minSalary, startDate) selected, show things that should be shown
```
Comment (cowmanjoe) [inline client/src/components/ResultRow.js:36]: Is the constructor supposed to be commented out?

File: client/src/components/ResultData.js [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
-    this.postUrl = postUrl;
-    this.applyUrl = applyUrl;
+    this.city = city;
+    this.state = state;
+    this.country = country;
+    this.latitude = latitude;
+    this.longitude = longitude;
+    this.companyName = companyName;
+    this.location = "";
+    this.minSalary = minSalary;
```
Comment (cowmanjoe) [inline client/src/components/ResultData.js:14]: Small thing but DB name is actually salary_min so maybe use salaryMin here?

File: client/src/components/ResultData.js [hunk 3]
```diff
@@ -1,10 +1,19 @@
 export default class ResultData {
-  constructor(title, description, excerpt, location, postUrl, applyUrl) {
+  constructor(title, link, description, excerpt, city, state, country, latitude, longitude, companyName, minSalary, startDate) {
```
Comment (cowmanjoe) [inline client/src/components/ResultData.js:2]: excerpt is not included in the database. We can generate this by shortening the description and appending an ellipsis to the end.

File: client/src/components/ResultRow.js (hunk 1 above)
Comment (cowmanjoe) [reply client/src/components/ResultRow.js:36]: Also the database names are not camelCase, they are snake_case. Check out the schema [here](https://github.com/ubclaunchpad/Internado/wiki/Proposed-Database-Schema) for all the details. I'm not sure whether the properties of ResultData should be snake case as well, but if not, they need to be translated somewhere.

**Review by cowmanjoe — CHANGES_REQUESTED** [review]
Good stuff. I changed the base to develop because I don't think we are using integration anymore. Requested a few changes, in general I think the main thing is to make sure that the exact schema is translated into the ResultData object somewhere before it is used. If this was meant to be included in our demo, branch off of feature/121 and make a PR into that. Thanks!

File: client/src/components/ResultData.js (hunk 3 above)
Comment (MichaelXian) [reply client/src/components/ResultData.js:2]: I just accidentally left this in

Comment (MichaelXian) [reply client/src/components/ResultData.js:2]: I've resolved these in the new branch I made, but I'm still testing it atm

Comment (cowmanjoe) [reply client/src/components/ResultData.js:2]: Great!

Comment (frostyshadows) [reply client/src/components/ResultData.js]: By the way, title should be job_title based on the most up-to-date version of the schema.

File: client/src/components/ResultRow.js [hunk 4]
```diff
@@ -31,9 +31,9 @@ class ResultRow extends Component {
     let arrow = '>';
     let originalPosting = '';
     let applyButton = '';
-    let { location } = this.props.result;
+    let location = this.props.result.city + ", " + this.props.result.state + " " + this.props.result.country;
```
Comment (frostyshadows) [inline client/src/components/ResultRow.js:34]: could you use a Javascript string template for this instead? They're a bit cleaner to read.

File: client/src/reducers/postingsReducer.js [hunk 5]
```diff
@@ -3,9 +3,22 @@ import ResultData from '../components/ResultData';
...
-    new ResultData("CPU maker", "Make CPUs and stuff. You also do things that make this string longer, so I can test how well this works for long strings. Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong ", "Make CPUs and stuff.",  "Vancouver, WA, USA", "https://ded.increpare.com/~locus/yourname.html", "https://ded.increpare.com/~locus/yourname.html"),
-    new ResultData("Software Developer", "Develop Software, but more text", "Develop Software", "Vancouver, BC, Canada", "https://github.com/mattdiamond/fuckitjs", "https://github.com/ajalt/fuckitpy")
+    new ResultData("Title",
```
Comment (frostyshadows) [inline client/src/reducers/postingsReducer.js:6]: Is this dummy data still used? If so, make sure it matches what's coming from the server. For example, latitude, longitude, and salary_min are numbers in the raw JSON from the server.
Here's what a job coming from the server should look like: https://github.com/ubclaunchpad/Internado/blob/develop/server/models/job.ts

**Review by frostyshadows — CHANGES_REQUESTED** [review]
A few more minor changes.

**Comment by cowmanjoe** [comment]
Should this be closed on account of #126 being opened?

**Pull request closed without merging**